### PR TITLE
Troubleshoot Test Code in Javascript

### PR DESCRIPTION
**Pull request by juan-fernandez**

### CODE REVIEW

File: docs/javascript-troubleshooting.md [hunk 1]
```diff
@@ -27,3 +27,48 @@ You can find further information in [Scope Javascript Agent - Running tests insi
...
+    /ui
+      cypress.json
+      /cypress
+        /integration
+          integration.spec.js
+    /admin-ui
+      cypress.json
+      /cypress
+        /smoke
+          smoke.spec.js
```
Comment (juan-fernandez) [inline docs/javascript-troubleshooting.md:47]: not sure if this is the best way to represent a folder tree structure

Comment (drodriguezhdez) [reply docs/javascript-troubleshooting.md:47]: Have you tried using a tool? E.g: https://www.npmjs.com/package/mddir
More info about it in [this post](https://stackoverflow.com/questions/19699059/representing-directory-file-structure-in-markdown-syntax)

Comment (juan-fernandez) [reply docs/javascript-troubleshooting.md:47]: cool! Thanks!

Comment (juan-fernandez) [reply docs/javascript-troubleshooting.md]: tried 👍 , much better

Comment (drodriguezhdez) [reply docs/javascript-troubleshooting.md]: awesome

**Pull request merged**